### ISSUE TITLE
cactus-graphmap-join error 

### ISSUE THREAD

**Issue by majocava** [ComparativeGenomicsToolkit/cactus#1123]
Hello,

I'm following the The Minigraph-Cactus Pangenome Pipeline and managed to complete almost all steps. However, when I ran the cactus-graphmap-join step with command cactus-graphmap-join ./jobstore --vg cpelliculosa_separate_pg/chrom-alignments/*.vg --hal cpelliculosa_separate_pg/chrom-alignments/*.hal \
--outDir ./cpelliculosa_separate_pg --outName cpelliculosa_separate_pg --reference GCF_0016612551 --vcf --giraffe clip , I get an error in the vg validate step. 

```
Traceback (most recent call last):
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/toil/worker.py", line 403, in workerScript
	    job._runner(jobGraph=None, jobStore=jobStore, fileStore=fileStore, defer=defer)
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/toil/job.py", line 2774, in _runner
	    returnValues = self._run(jobGraph=None, fileStore=fileStore)
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/toil/job.py", line 2691, in _run
	    return self.run(fileStore)
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/toil/job.py", line 2919, in run
	    rValue = userFunction(*((self,) + tuple(self._args)), **self._kwargs)
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/cactus/refmap/cactus_graphmap_join.py", line 596, in clip_vg
	    cactus_call(parameters=['vg', 'validate', clipped_path])
	  File "/home/lab/cactus-bin-v2.6.5/cactus_env/lib/python3.10/site-packages/cactus/shared/common.py", line 889, in cactus_call
	    raise RuntimeError("{}Command {} exited {}: {}".format(sigill_msg, call, process.returncode, out))
	RuntimeError: Command ['vg', 'validate', '/tmp/5123fb3e24865a30a8e57aade77df55d/9605/f610/tmp_labms3j/NW_017567117.1.vg.clip'] exited 1: stderr=graph invalid: missing edge between 652th step (64036:0) and 653th step (64036:1) of path GCA_0193217651#0#JAHTLY010002216.1#0
	graph invalid: missing edge between 653th step (64036:0) and 652th step (64036:0) of path GCA_0193217651#0#JAHTLY010002216.1#0
	graph: invalid
	
	[2023-08-03T11:40:10-0500] [MainThread] [E] [toil.worker] Exiting the worker because of a failed job on host omics
```

[Log_error.txt](https://github.com/ComparativeGenomicsToolkit/cactus/files/12253390/Log_error.txt)

**Comment by glennhickey** [comment]
This looks like a bug.  Are you able to share the input `.vg` files to `cactus-graphmap-join`?

**Comment by majocava97** [comment]
These are the files: 
https://drive.google.com/drive/folders/1fwbkMi2h4xNyrQYykv9LdKv0B_Ez6Nrf?usp=drive_link

**Comment by glennhickey** [comment]
Hi, thanks for sharing the data.  I get a different error when trying to run your command (empty intermediate file) and can't reproduce your error.  Could you please share the original input fasta files?  I'll try running the whole thing from scratch which will hopefully be more productive.  thanks!

**Comment by majocava97** [comment]
Hi. Thank you for your response. I've uploaded the sequences here as well as the txt file. Thank you 

https://drive.google.com/drive/folders/1fwbkMi2h4xNyrQYykv9LdKv0B_Ez6Nrf?usp=drive_link

**Comment by glennhickey** [comment]
Got them.  There are 17 fasta files and only 16 in the text file.  Is that expected?  Also, what is the reference here?

**Comment by majocava97** [comment]
I only ran the 16 fasta files specified in the text file.
The reference is GCF_0016612551.

**Comment by majocava** [comment]
Thank you so much! I was able to run the pipeline successfully.

**Comment by mjhubisz** [comment]
Hi,
This bug in gfaffix popped back up (see recent updates here: https://github.com/marschall-lab/GFAffix/issues/11). They just released a new version with a fix, v0.1.5b. I am wondering if we can get the updated GFAffix incorporated into a cactus release? Thank you!

**Comment by glennhickey** [comment]
Thanks. It's already on its way into Cactus via #1192

Hoping to make a release this week. Am waiting for a test to go through on a very busy cluster first.
